src/primitives.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Event names that can occur in this application.
+ * Event names that can occur in the applications.
```

Comment:
super grammar nit but feels weird as is
```suggestion
 * Event names that can occur in an application.
```